src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -399,7 +399,14 @@ public void close(Collection<TopicPartition> partitions) {
   }
 
   @Override
-  public void setIsStoppedToTrue() {}
+  public void stop() {
+    try {
+      streamingIngestClient.close();
+    } catch (Exception e) {
+      LOGGER.warn("Could not close streaming ingest client");
```

Comment:
Do we still need the log if we're throwing an exception?

Reply:
Exception removed.